pkgs/development/python-modules/apache-airflow/default.nix

```diff
@@ -126,7 +126,7 @@ buildPythonPackage rec {
     flask-appbuilder
     flask-caching
     flask_login
-    flask_wtf
+    flask_wtf.wtforms2
```

Comment:
airflow is so often broken, I don't really care anymore at this point. I would suggest to just leave it like it is and let someone else care about it.